test/travis.bash

```diff
@@ -42,31 +42,40 @@ esac
 
 make
 bin/ch-run --version
-version=$(cat VERSION.full)
 
 if [[ $INSTALL ]]; then
     sudo make install PREFIX="$PREFIX"
-    cd "$PREFIX/libexec/charliecloud-$version"
+    ch_test="${PREFIX}/bin/ch-test"
+else
+    ch_test=$(readlink -f bin/ch-test)
```

Comment:
Removing the absolute path here will cause travis to error (https://travis-ci.org/hpc/charliecloud/jobs/604006996?utm_medium=notification&utm_source=email). 

```bash
147 top_dir=$(readlink -f "${ch_bin%/*}")
[...]
173 # Determine source vs prefix installation.
174 if [[ -d "${top_dir}/test" ]]; then
175     # source install
176     testdir=${top_dir}/test
177 else
178     # prefix installation
179     testdir=${top_dir%/charliecloud*}/libexec/charliecloud-$(version 2>&1)/test
180 fi
```


The check for `${top_dir}/test` fails (it shouldn't, it is a source install). I feel like this is because `base.sh` uses `dirname "$0"` instead of `dirname "${BASH_SOURCE[0]}`. When we do `ch_test=$(readlink -f bin/ch-test` in the `travis.bash`, the script the problem is resolved (https://travis-ci.org/hpc/charliecloud/jobs/604009862?utm_medium=notification&utm_source=github_status).